### PR TITLE
LGTM fixes

### PR DESCRIPTION
**Pull request by ramimac**
# Description

Made trivial fixes to LGTM issues identified (https://lgtm.com/projects/g/nccgroup/ScoutSuite/alerts/?mode=list) to improve code quality

## Type of change

Select the relevant option(s):

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (optional)
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tools/process_raw_response.py [hunk 1]
```diff
@@ -59,10 +58,7 @@ def camel_to_snake(name, upper=False):
 
     if args.provider == 'aws':
         object_format = 'raw_{}.get(\'{}\')'
-        cleaned_value = args.value.replace('<class \'dict\'>: ', '')
-        cleaned_value = args.value.replace('\}', '}')
-        cleaned_value = cleaned_value.replace(", tzinfo=tzlocal()", "")
-        cleaned_value = cleaned_value.replace(", tzinfo=tzutc()", "")
+        cleaned_value = args.value.replace('<class \'dict\'>: ', '').replace('\}', '}').replace(", tzinfo=tzlocal()", "").replace(", tzinfo=tzutc()", "")
```
Comment (j4v) [inline tools/process_raw_response.py:61]: Better keep these separate, it's not clean and we should really use regexes, but this way seems worst.

Comment (ramimac) [reply tools/process_raw_response.py:61]: Will revert per-your preference, IMHO they are equally legible and this way is "technically correct" though

File: tools/process_raw_response.py [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 import argparse
 import json
-import datetime
```
Comment (j4v) [inline tools/process_raw_response.py:5]: While this is flagged by LGTM, it is required for the `eval` on L65.

Comment (ramimac) [reply tools/process_raw_response.py]: weird, sorry for that then, datetime is back in

**Pull request merged**